src/main/java/com/gluonhq/substrate/util/ios/Deploy.java

```diff
@@ -162,11 +162,15 @@ public boolean install(String app) throws IOException, InterruptedException {
             boolean result = runner.runTimedProcess("run", 60);
             Logger.logInfo("result = " + result);
             if (result) {
-                if (runner.getResponses().stream().anyMatch(l -> "Error: The device is locked.".equals(l))) {
-                    Logger.logInfo("Device locked! Please, unlock and press ENTER to try again");
+                if (runner.getResponses().stream().anyMatch("Error: The device is locked."::equals)) {
+                    Logger.logInfo("\n\nDevice locked!\nPlease, unlock and press ENTER to try again");
                     System.in.read();
                     keepTrying = true;
                 }
+                if (runner.getResponses().stream().anyMatch(l -> l.contains("Assertion failed: (AMDeviceIsPaired(device))"))) {
```

Comment:
Is there a way we can detect this before trying to upload the app?

Reply:
Running `ios-deploy -C` fails if computer is not trusted:
```
iios-deploy -C
[....] Waiting for iOS device to be connected
[....] Using a98438b083503e27713fa4863XXXXX (D221AP, iPhone X, iphoneos, arm64) a.k.a. 'iPhone'.
Assertion failed: (AMDeviceIsPaired(device)), function get_battery_level, file /tmp/ios-deploy-20200102-40963-7r6yxb/src/ios-deploy/ios-deploy.m, line 1356.
zsh: abort      ios-deploy -C
```
If the computer is trusted:

```
ios-deploy -C
[....] Waiting for iOS device to be connected
[....] Using a98438b083503e27713fa4863XXXXX (D221AP, iPhone X, iphoneos, arm64) a.k.a. 'iPhone.
BatteryCurrentCapacity:79

```

Should be easy to test in advance, right after checking that `ios-deploy` is installed?

Reply:
I've pushed that change, it will fail fast now if the computer is not trusted.